deploy/__main__.py

```diff
@@ -109,6 +110,10 @@ def main():
         build()
     elif args.action == "deploy":
         deploy(args.cluster_name, args.hub_name, args.skip_hub_health_test)
+    else:
+        # FIXME: Not sure how to achieve this with just argparse?
+        print(argparser.format_help(), file=sys.stderr)
```

Comment:
Is this to print a help message when no arguments are passed to the script?

I noticed that when you pass it anything other than `build/deploy`, it prints a `usage` help message. But when you run it with no arguments, it  just exits and doesn't print anything.